src/main/webapp/index.html

```diff
@@ -6,9 +6,12 @@
     <link href="style.css" rel="stylesheet">
 </head>
 <div>
-    <div id="left-widget"></div>
-    <form id="task-list"></form>
-    <button onclick="schedule()">Schedule!</button>
+    <div id="left-widget">
```

Comment:
This file uses 4 spaces indent instead of 2.

Reply:
Thanks!